From 4110fa30bb20d5912c9bdbffde1dc32d2fc10937 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Mon, 27 Jul 2020 09:26:29 -0700
Subject: [PATCH 1469/1851] drm: xlnx: pl_disp: remove unused variables

commit df5d3c56b8f18bbe1bc18a25d7312d01c53ce719 from
https://github.com/Xilinx/linux-xlnx.git

drivers/gpu/drm/xlnx/xlnx_pl_disp.c: In function 'xlnx_pl_disp_plane_atomic_check':
drivers/gpu/drm/xlnx/xlnx_pl_disp.c:317:31: warning: variable 'old_crtc_state' set but not used [-Wunused-but-set-variable]
  317 |  const struct drm_crtc_state *old_crtc_state;
      |                               ^~~~~~~~~~~~~~

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_pl_disp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_pl_disp.c b/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
index f5f1ce64266d..8f810729d01a 100644
--- a/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
+++ b/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
@@ -314,13 +314,11 @@ xlnx_pl_disp_plane_atomic_check(struct drm_plane *plane,
 	const struct drm_plane_state *old_plane_state =
 		drm_atomic_get_old_plane_state(state, plane);
 	struct drm_crtc *crtc = new_plane_state->crtc ?: old_plane_state->crtc;
-	const struct drm_crtc_state *old_crtc_state;
 	struct drm_crtc_state *new_crtc_state;
 
 	if (!crtc)
 		return 0;
 
-	old_crtc_state = drm_atomic_get_old_crtc_state(state, crtc);
 	new_crtc_state = drm_atomic_get_new_crtc_state(state, crtc);
 
 	/* plane must be enabled when state is active */
-- 
2.31.1

